marketplace/core/types/channels/whatsapp_demo/views.py

```diff
@@ -28,20 +28,16 @@ def perform_create(self, serializer):
             facebook_access_token="null",
         )
 
-        task = celery_app.send_task(
-            name="create_channel", args=[user.email, str(instance.project_uuid), data, instance.channeltype_code]
-        )
-        task.wait()
-
-        result = task.result
+        client = ConnectProjectClient()
+        result = client.create_channel(user.email, str(instance.project_uuid), data, instance.channeltype_code)
 
         instance.config["title"] = result.get("name")
         instance.config["channelUuid"] = result.get("uuid")
 
-        task = celery_app.send_task(name="get_channel_token", args=[result.get("uuid"), result.get("name")])
-        task.wait()
+        ch_client = WPPRouterChannelClient()
```

Comment:
Call this variable `channel_client` to improve readability

Reply:
done!